tests/subsys/settings/functional/nvs/prj_native_posix.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_ZTEST=y
```

Comment:
Please create a `boards` directory and move board specific config in to this directory and keep one prj.conf file

Reply:
done, thanks for the tip